hydroflow/examples/kvs_bench/client.rs

```diff
@@ -0,0 +1,79 @@
+use crate::{KVSRequest, KVSResponse};
+use futures::SinkExt;
+use hydroflow::util::{deserialize_from_bytes, serialize_to_bytes, tcp_bytes};
+use rand::{distributions::Uniform, prelude::Distribution, rngs::StdRng, RngCore, SeedableRng};
+use std::{
+    collections::HashMap,
+    net::SocketAddr,
+    time::{Duration, Instant},
+};
+use tokio::net::TcpStream;
+use tokio_stream::StreamExt;
+
+pub async fn run_client(server_addr: SocketAddr) {
+    println!("tid client: {}", palaver::thread::gettid());
+
+    let stream = TcpStream::connect(server_addr).await.unwrap();
+    stream.set_nodelay(true).unwrap();
+
+    println!("connected");
+
+    let (mut outbound, mut inbound) = tcp_bytes(stream);
```

Comment:
So one thing, I'm not sure if the Anna KVS benchmark even exercised the networking stack >.>
May be an unfair comparison that way